ansible/roles/debops.prosody/README.md

```diff
@@ -0,0 +1,24 @@
+## [![DebOps project](http://debops.org/images/debops-small.png)](http://debops.org) prosody
```

Comment:
The `README` files in individual roles aren't currently needed since the roles are not published separately on GitHub anymore. Role documentation should be the go-to place for information about the roles themselves.

Reply:
k, deleted